exporter/sumologicexporter/config.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sumologicexporter
+
+import (
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// Config defines configuration for Sumo Logic exporter.
+type Config struct {
+	configmodels.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
+
+	// Global options
+	// Unique URL generated for your HTTP Metrics Source.
+	// This is the address to send metrics to.
+	// url: "https://events.sumologic.net/receiver/v1/http/<UniqueHTTPCollectorCode>"
+	URL string `mapstructure:"url"`
+	// Timeout used for HTTP request
+	Timeout int32 `mapstructure:"timeout"`
+	// HTTP method, one of: "POST" or "PUT". "POST" is used by default if unset.
+	Method string `mapstructure:"method"`
+	// Option to enable compression (default false)
+	Compress bool `mapstructure:"compress"`
+	// Compression encoding format, either gzip or deflate (default gzip)
+	CompressEncoding string `mapstructure:"compress_encoding"`
+	// Max HTTP request body size in bytes before compression (if applied).
+	// By default 1MB is recommended.
+	// NOTE:
+	// Bear in mind that in some serializer a metric even though serialized to multiple
+	// lines cannot be split any further so setting this very low might not work
+	// as expected.
+	MaxRequestBodySize int64 `mapstructure:"max_request_body_size"`
+
+	// Logs related configuration
+	// Format to post logs into Sumo. (default json)
+	//   * text - Logs will appear in SumoLogic in text format (taken from the field specified in log_key)
+	//   * json - Logs will appear in SumoLogic in json format.
```

Comment:
```suggestion
	//   * text - Logs will appear in Sumo Logic in text format (taken from the field specified in log_key)
	//   * json - Logs will appear in Sumo Logic in json format.
```

We've already seen Stefan correct it :) https://github.com/SumoLogic/sumologic-kubernetes-tools/commit/73bd1d465148869392db8c678949a0d5696a4aec